src/poojap/Assignment8.java

```diff
@@ -0,0 +1,65 @@
+package poojap;
+
+public class Assignment8 {
+	
+void evenNumber(int start,int end) {
+		
+		int i=start;
```

Comment:
You can handle this scenario without creating third variable, think on this.
can you use start++ for increment?